testing/matrix-sdk-test/src/test_json/sync.rs

```diff
@@ -1349,3 +1349,67 @@ pub static VOIP_SYNC: Lazy<JsonValue> = Lazy::new(|| {
         }
     })
 });
+
+pub static PARENT_SPACE_SYNC: Lazy<JsonValue> = Lazy::new(|| {
```

Comment:
Can you move this to the spaces module? We're trying to get away from JSON blobs being defined far away from the tests that use them.